### ISSUE TITLE
SASS globbing appears broken in v4.0.0.rc1

### ISSUE THREAD

**Issue by lachlanmcdonald** [middleman/middleman#1627]
I've been porting one of our frameworks over to v4.0.0.rc1 (just to see how much we have to refactor) and identified a change in behaviour. In 3.4 you could glob multiple SASS files using:

```sass
@import "test/**/*";
```

And everything under the `test` folder would be imported without a hitch. On 4.0.0.rc1 the same code throws the following error:

```
Sass::SyntaxError: File to import not found or unreadable: test\**\*.
Load paths:
  \Users\lacmcdonald\Sites\mmtest\source\javascripts
  \Users\lacmcdonald\Sites\mmtest\source\stylesheets
```

To test, I ran `middleman init` and and moved:

- `stylesheets/site.css` to `stylesheets/test/_site.scss`
- `stylesheets/normalise.css` to `stylesheets/test/_normalise.scss`

Then created a new `site.css.scss` file with the above `@import`. I opted in to the Asset Pipeline and Compass during the init.

Might have just done something stupid during the init, but globbing is a pretty big win for us with some projects spread across hundreds of partials, so hope we can find a solution.

**Comment by tdreyno** [comment]
@lachlanmcdonald Do you have `middleman-sprockets` in your `Gemfile.lock`?

Can you point your gems at `master`? This might have already been fixed.

**Comment by lachlanmcdonald** [comment]
Here are the contents of the `Gemfile`:
```
source 'https://rubygems.org'
gem 'wdm', '~> 0.1.0', platforms: [:mswin, :mingw]
gem 'tzinfo-data', platforms: [:mswin, :mingw, :jruby]
gem 'middleman-sprockets', '>= 4.0.0.rc.1'
gem 'middleman-compass', '>= 4.0.0.rc.1'
gem 'middleman', '>= 4.0.0.rc.1'
gem 'middleman-livereload'
```

**Comment by tdreyno** [comment]
@lachlanmcdonald If you remove `middleman-compass`, do you have the same issues?

**Comment by lachlanmcdonald** [comment]
@tdreyno Same error persists.

Here's a copy of the full output, `Gemfile` and `Gemfile.lock`:
https://gist.github.com/lachlanmcdonald/40a2c688e5c34dd9f692

**Comment by tdreyno** [comment]
Great, thank you. Will resolve before next RC

**Comment by johnnyshields** [comment]
PR to fix this issue is here: https://github.com/middleman/middleman/pull/1695

**Comment by johnnyshields** [comment]
@tdreyno you can close this one.

**Comment by robertwbradford** [comment]
I'm still seeing the same problem. I created a sample app showing the error when you start the server and try to load the page:

https://github.com/robertwbradford/middleman-with-sass-globbing

Perhaps I'm confused over which gem versions I should be using. This is a snippet from my Gemfile.lock for that app:
```
    middleman (4.1.2)
      coffee-script (~> 2.2)
      compass-import-once (= 1.0.5)
      haml (>= 4.0.5)
      kramdown (~> 1.2)
      middleman-cli (= 4.1.2)
      middleman-core (= 4.1.2)
      sass (>= 3.4.0, < 4.0)
    middleman-cli (4.1.2)
      thor (>= 0.17.0, < 2.0)
    middleman-compass (4.0.1)
      compass (>= 1.0.0, < 2.0.0)
      middleman-core (>= 4.0.0)
    middleman-core (4.1.2)
      activesupport (~> 4.2)
      addressable (~> 2.4.0)
      backports (~> 3.6)
      bundler (~> 1.1)
      capybara (~> 2.5.0)
      contracts (~> 0.13.0)
      erubis
      execjs (~> 2.0)
      fast_blank
      fastimage (~> 1.8)
      hamster (~> 3.0)
      hashie (~> 3.4)
      i18n (~> 0.7.0)
      listen (~> 3.0)
      padrino-helpers (~> 0.13.0)
      parallel
      rack (>= 1.4.5, < 2.0)
      sass (>= 3.4)
      tilt (~> 1.4.1)
      uglifier (~> 2.6)
    middleman-livereload (3.4.6)
      em-websocket (~> 0.5.1)
      middleman-core (>= 3.3)
      rack-livereload (~> 0.3.15)
    middleman-sprockets (4.0.0.rc.1)
```
What am I missing?

**Comment by stevenosloan** [comment]
Took me a while to track that down -- is middleman-sprockets related. With 4.0.x it dropped sprockets-sass as a dependency (where the globbing was coming from).

With your example repo, if you add:

```ruby
gem 'sass',          '~> 3.4'
gem 'sass-globbing', '~> 1.1'
```
You should be good.

**Comment by tdreyno** [comment]
Thanks @stevenosloan

**Comment by robertwbradford** [comment]
Thank you, @stevenosloan!

Even after doing this, I was still getting an error with it complaining about not being able to find two of the files. After some more digging, I found that these two files to import were `.css` files, not `.scss` files. It looks like we now have to be explicit about `.css` files? For example, this used to work (with both files being `.css` files:

```
@import "lib/resets/normalize";
@import "lib/resets/site";
```
However, this now breaks and the following is required:
```
@import "lib/resets/normalize.css";
@import "lib/resets/site.css";
```
I added a comment here: https://github.com/middleman/middleman-sprockets/issues/62#issuecomment-196897018
